### DISCUSSION TITLE
Make web page full width when using code editor

### DISCUSSION THREAD

**Discussion by TiBiBa** [hedyorg/hedy#3003] [Website]
**Description**
Our UI is getting crowed. The actual part for coding is small and we "lose" a lot of space due to the menubar, save/share bar and the adventures tabs. These are nice and belong to our UI, but they take a way a lot of space from our actual code editor and output window. I suggest to completely re-style the "hedy" page, making more space for the actual coding parts.

Not sure on a direct approach (yet). But I suggest to make multiple concepts and discuss on all to decide on the best approach. We do have to keep a lot of aspects in mind, for example the interactive tutorial, responsive design and auto-selecting of tabs when loading a saved program.

**Idea**
- Change the container structure when on "Hedy" page
- Restyle UI, make it less crowded
- Minimize adventures by default
- Make a dedicated window for "variable view"
- Make code editor and out window wider

**Comment by sanyaade-teachings** [comment]
Just saw this being raised here. I was actually think of this when I visited the "Hedy" page
Make ==> story, adventure, etc... enable at certain levels (prevent kids jumping into confusing levels and menus)

I will need to customise Hedy for my new teaching classes

God blesses!!!

Regards
